docs/source/examples/LCS_ex.py

```diff
@@ -0,0 +1,24 @@
+#creating a linked model
+from gpkit import Variable, Model, LinkedConstraintSet, ConstraintSet
+from import_classes import M1, M2
+
+#Create the two models
+m1 = M1()
+m2 = M2()
+
+#create a list of submodes
+submodels = [m1, m2]
+
+#generate a constraint set for the full model
+constraints = ConstraintSet([submodels])
+
+#create a linked constraint set
+#now there is only a single y variable
+lc = LinkedConstraintSet(constraints)
```

Comment:
I'd simplify by deleting two lines above this one and changing this one to `lc = LinkedConstraintSet([m1, m2])`.